projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/welcome/HelloTest.java

```diff
@@ -0,0 +1,44 @@
+package xxxxxx.yyyyyy.zzzzzz.selenium.welcome;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * Executes the test for the application home page.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "classpath:META-INF/spring/seleniumContext.xml" })
+public class HelloTest {
+
```

Comment:
`WelcomePageTest` or `WelcomeTest` would be better according to `/` and `WelcomeConteoller`